### DISCUSSION TITLE
The 1.1.3 version of the panel cannot change the default language

### DISCUSSION THREAD

**Discussion by wuzhaochun** [pterodactyl/panel#2897] [Feature Requests]
This is my own Chinese language document. After the file is put into the "Lang" directory, the default language of the Administrator Console cannot be changed.
[zh.zip](https://github.com/pterodactyl/panel/files/5727733/zh.zip)

**Comment by DaneEveritt** [comment]
This is already known and intentional. There are multiple issues that have been opened about it.
